x/superfluid/keeper/slash_test.go

```diff
@@ -45,8 +45,8 @@ func (s *KeeperTestSuite) TestBeforeValidatorSlashed() {
 			[]stakingtypes.BondStatus{stakingtypes.Unbonded},
 			1,
 			[]superfluidDelegation{{0, 0, 0, 1000000}},
-			[]int64{0},
-			[]int64{0},
+			[]int64{},
+			[]int64{}, // UNFORKING v2 TODO: We no longer can slash unbonded validators, verify that this is correct
```

Comment:
.

Reply:
This logic branch is new to v0.50:

https://github.com/osmosis-labs/cosmos-sdk/blob/e899ad66a8bc30c82a2f2c0bb00a9fce6bf4ea0a/x/staking/keeper/slash.go#L70-L73

It makes sense, because an unbonded validator should not be able to be slashed. Its just this logic was not there previously, and we had test cases that tested if unbonded validtors would be slashed, which wouldn't happen on mainnet, but we tested anyway.